loader.lua

```diff
@@ -590,7 +592,9 @@ local function queryCard(cardID, forceNameQuery, forceSetNumLangQuery, onSuccess
 
     local language_code = getLanguageCode()
 
-    if forceNameQuery then
+    if getFirstPrint then
```

Comment:
I think we should keep `forceNameQuery` as the first if case. That only gets set when the normal query method doesn't succeed, so its important that its always first.